TST: test_transform_bounds__ignore_inf (PROJ Latest)

https://github.com/pyproj4/pyproj/actions/runs/3814386425/jobs/6488655015
```
FAILED test/test_transformer.py::test_transform_bounds__ignore_inf[ESRI:102036-expected_bounds0] - AssertionError: 
Arrays are not almost equal to 0 decimals

Mismatched elements: 4 / 4 (100%)
Max absolute difference: 1.2120748e+09
Max relative difference: 12.5916335
 x: array([-1.e+09, -1.e+09,  1.e+09,  1.e+09])
 y: array([        0, -89178008,         0,         0])
```
```
<Projected CRS: ESRI:102036>
Name: South_Pole_Gnomonic
Axis Info [cartesian]:
- E[east]: Easting (metre)
- N[north]: Northing (metre)
Area of Use:
- name: Southern hemisphere.
- bounds: (-180.0, -90.0, 180.0, 0.0)
Coordinate Operation:
- name: South_Pole_Gnomonic
- method: Gnomonic
Datum: World Geodetic System 1984
- Ellipsoid: WGS 84
- Prime Meridian: Greenwich
```

Related: https://github.com/OSGeo/PROJ/pull/3522

--- a/test/test_transformer.py
+++ b/test/test_transformer.py
@@ -21,6 +21,7 @@ from pyproj.transformer import AreaOfInterest, TransformerGroup
 from test.conftest import (
     PROJ_GTE_9,
     PROJ_GTE_91,
+    PROJ_GTE_92,
     grids_available,
     proj_env,
     proj_network_env,
@@ -1367,17 +1368,21 @@ def test_transform_bounds__beyond_global_bounds():
 
 
 @pytest.mark.parametrize(
-    "input_crs,expected_bounds",
+    "input_crs,input_bounds,expected_bounds",
     [
-        ("ESRI:102036", (0, -89178008, 0, 0)),
-        ("ESRI:54026", (0, -179545824, 0, 179545824)),
+        (
+            "ESRI:102036",
+            (-180.0, -90.0, 180.0, 1.3 if PROJ_GTE_92 else 0),
+            (0, -116576599 if PROJ_GTE_92 else -89178008, 0, 0),
+        ),
+        ("ESRI:54026", (-180.0, -90.0, 180.0, 90.0), (0, -179545824, 0, 179545824)),
     ],
 )
-def test_transform_bounds__ignore_inf(input_crs, expected_bounds):
+def test_transform_bounds__ignore_inf(input_crs, input_bounds, expected_bounds):
     crs = CRS(input_crs)
     transformer = Transformer.from_crs(crs.geodetic_crs, crs, always_xy=True)
     assert_almost_equal(
-        transformer.transform_bounds(*crs.area_of_use.bounds),
+        transformer.transform_bounds(*input_bounds),
         expected_bounds,
         decimal=0,
     )

